### PR TITLE
[ui] Remove individuals affiliations

### PR DESCRIPTION
**Pull request by evamillan**
This PR adds the `withdraw` Apollo mutation, which removes an individual's affiliation to an organization.
An individual's affiliation can be removed by clicking the delete button next to it on `ExpandedIndividual`.

### CODE REVIEW

File: ui/src/apollo/mutations.js [hunk 1]
```diff
@@ -384,6 +419,17 @@ const deleteDomain = (apollo, domain) => {
   return response;
 };
 
+const withdraw = (apollo, uuid, organization) => {
```
Comment (sduenas) [inline ui/src/apollo/mutations.js:422]: This mutation must also get the period of time to remove. Remember that an Individual could be affiliated to the same organization during different periods of time.

**Review by sduenas — CHANGES_REQUESTED** [review]
Please check my comment. The rest of the PR looks good to me.

**Comment by evamillan** [comment]
Thank you, I updated the PR.

**Review by sduenas — APPROVED** [review]
LGTM

**Pull request merged**